.github/workflows/tests.yaml

```diff
@@ -113,39 +114,43 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Conda Setup
-        uses: s-weigand/setup-conda@v1
+        uses: conda-incubator/setup-miniconda@v3
         with:
-          conda-channels: conda-forge
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          use-mamba: true
+          auto-update-conda: true
+          use-only-tar-bz2: false
```

Comment:
You can use `activate-environment: test` here and remove the activations from all `run` commands.